source/agora/test/Base.d

```diff
@@ -1597,7 +1600,7 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
         {
             banman : ban_conf,
             node : makeNodeConfig(self_address),
-            validator : ValidatorConfig(true, key_pair, [self_address]),
+            validator : ValidatorConfig(true, key_pair, [self_address], "", test_conf.recurring_enrollment),
```

Comment:
I've just noticed that `registry_address` from `TestConf` is not used. That's a bug.